INSTALL.rst

```diff
@@ -7,14 +7,15 @@ B2Share installation
 ===============================================
 
 
-1.0. Prerequisite: clone B2SHARE
---------------------------------
+1.0. Prerequisite: clone `dockerize`
+------------------------------------
 
-Current installation process requires a local clone of B2SHARE.
+The current installation process requires a local clone of the `dockerize`
```

Comment:
The docker image can be used separately. Thus I would rather write something like this:

===

The b2share docker image can be used as is and is available on dockerhub as [`eudatb2share/b2share`](https://hub.docker.com/r/eudatb2share/b2share/). However it requires many different services like elasticsearch, postgresql, redis and rabbitmq, thus we provide a standard deployment procedure which creates a demonstration environment.
The next steps will describe how to use this deployment.

First clone the `dockerize` repository:

===